infra/vsts_agent_linux_startup.sh

```diff
@@ -28,7 +28,7 @@ apt-get install -qy \
   git \
   netcat
 
-curl -sSO https://dl.google.com/cloudagents/install-logging-agent.sh | bash
+curl -sS https://dl.google.com/cloudagents/install-logging-agent.sh | bash
```

Comment:
```suggestion
curl -sSL https://dl.google.com/cloudagents/install-logging-agent.sh | bash
```
Following redirects is usually a reasonable choice.

Reply:
Agreed ^ I was going to suggest adding `-L` :-)